deno.json

```diff
@@ -0,0 +1,6 @@
+{
+  "lock": false,
+  "name": "@bartlomieju/postgres",
```

Comment:
Shouldnt this be `denodrivers/postgres`?

Reply:
@bartlomieju wanted it under his own scope.